main/settings_celery.py

```diff
@@ -6,6 +6,16 @@
 
 from main.envs import get_bool, get_int, get_string
 
+"""
+the lookback window (in minutes) for the embeddings task
+the frequency of the embeddings tasks will be 1/4 of this for example
+lookback = 2 hours; task frequency = 30 minutes
+"""
+QDRANT_EMBEDDINGS_TASK_LOOKBACK_WINDOW = get_int(
+    name="QDRANT_EMBEDDINGS_TASK_LOOKBACK_WINDOW", default=60 * 2
+)
+EMBEDDING_SCHEDULE_MINUTES = int(QDRANT_EMBEDDINGS_TASK_LOOKBACK_WINDOW / 4)
```

Comment:
So this is a nitpick but probably a fixed difference between QDRANT_EMBEDDINGS_TASK_LOOKBACK_WINDOW 
 and EMBEDDING_SCHEDULE_MINUTES  makes more sense than a ratio since the concern is how long celery takes to pick up tasks and that is the same whether the task runs every 30 minutes or every 3 hours. Also when we later decide that we are comfortable with a smaller difference between QDRANT_EMBEDDINGS_TASK_LOOKBACK_WINDOW and EMBEDDING_SCHEDULE_MINUTES because we have a task that picks up any missed embeddings  it makes more sense to think of that as a number of minutes than a ratio as well

Reply:
pushed a change to make the lookback window a fixed 60 minutes more than the schedule frequency.